.github/workflows/pull_request.yml

```diff
@@ -1,6 +1,14 @@
 name: Build Chain
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
```

Comment:
Not sure whether `.gitignore` should be excluded from checking. I think a change on it could break the buid of the repo. For example when committing a change on it plus a change spreaded on some files that are ignored and others that are not. Part of the change would be missing then.

Reply:
My previous comment doesn't make sense as a change in other files along `.gitignore` would trigger the build. Please ignore it.